common/src/Model/ChangeBrushFaceAttributesRequest.cpp

```diff
@@ -31,170 +31,171 @@
 
 namespace TrenchBroom {
     namespace Model {
-        template <typename T>
-        T evaluateValueOp(const T oldValue, const T newValue, const ChangeBrushFaceAttributesRequest::ValueOp op) {
-            switch (op) {
-                case ChangeBrushFaceAttributesRequest::ValueOp_Set:
-                    return newValue;
-                case ChangeBrushFaceAttributesRequest::ValueOp_Add:
-                    return oldValue + newValue;
-                case ChangeBrushFaceAttributesRequest::ValueOp_Mul:
-                    return oldValue * newValue;
-                case ChangeBrushFaceAttributesRequest::ValueOp_None:
-                    return oldValue;
-                    switchDefault()
-            }
-        }
-
-        template <typename T>
-        T evaluateFlagOp(const T oldValue, const T newValue, const ChangeBrushFaceAttributesRequest::FlagOp op) {
-            switch (op) {
-                case ChangeBrushFaceAttributesRequest::FlagOp_Replace:
-                    return newValue;
-                case ChangeBrushFaceAttributesRequest::FlagOp_Set:
-                    return oldValue | newValue;
-                case ChangeBrushFaceAttributesRequest::FlagOp_Unset:
-                    return oldValue & ~newValue;
-                case ChangeBrushFaceAttributesRequest::FlagOp_None:
-                    return oldValue;
-                    switchDefault()
+        namespace {
+            template <typename T>
+            T evaluateValueOp(const T oldValue, const T newValue, const ChangeBrushFaceAttributesRequest::ValueOp op) {
```

Comment:
I'd use `static` on all of these for readability, otherwise it's not obvious unless you scroll up/down to see the `namespace {`

Reply:
Ok!